update.sh

```diff
@@ -17,7 +17,7 @@ echo "Updating certificates..."
 ./certs/update.sh
 
 echo "Fetching and building traefik $VERSION..."
-wget -O traefik https://github.com/emilevauge/traefik/releases/download/$VERSION/traefik
+wget -O traefik https://github.com/containous/traefik/releases/download/$VERSION/traefik
```

Comment:
Darn, I read `curl` instead of `wget`. My bad, sorry for the confusion!

(And I didn't know that wget returns a proper return code on error, so that's good to know.)